tests/Endpoints/SearchTest.php

```diff
@@ -139,6 +139,100 @@ public function testExceptionIfNoIndexWhenSearching(): void
         $index->search('prince');
     }
 
+    public function testParametersCropMarker(): void
+    {
+        $response = $this->index->search('blood', [
+            'limit' => 1,
+            'attributesToCrop' => ['title'],
+            'cropLength' => 2,
+        ]);
+
+        $this->assertArrayHasKey('_formatted', $response->getHit(0));
```

Comment:
Because I was inspired by the tests already present and when I rewrite parts of them I use the getter. Or maybe sometimes I forgot that getter exist ... I will change it

Reply:
I prefer getters! Maybe it is worth creating an issue to change this consistently throughout the tests.

Reply:
The `rawSearch` is here to access your specific fields in `results` I find it really useful, I used it in tests when I need to check the exact field of it. Don't think it's not consistent.

Reply:
But we can create a specific class for the `movies` result or the `books` result in test

Reply:
The `rawResult` gives me access to the subfields of `_formatted` because it is an array.I used it for check the `cropMarker`